libglusterfs/src/common-utils.c

```diff
@@ -5426,3 +5436,26 @@ gf_nanosleep(uint64_t nsec)
 
     return ret;
 }
+
+int
+gf_syncfs(int fd)
+{
+    int ret = 0;
+#if defined(HAVE_SYNCFS)
+    /* Linux with glibc recent enough. */
+    ret = syncfs(fd);
+#elif defined(HAVE_SYNCFS_SYS)
+    /* Linux with no library function. */
+    ret = syscall(SYS_syncfs, fd);
+#else
+    /* Fallback to generic UNIX stuff. */
+    sync();
+#endif
+    return ret;
+}
+
```

Comment:
This doesn't seem to be a part of the devel patch...

Reply:
done